package.json

```diff
@@ -63,7 +63,7 @@
     "url-loader": "0.5.7",
     "webpack": "1.12.15",
     "webpack-bundle-size-analyzer": "2.0.2",
-    "webpack-dev-server": "1.14.1"
+    "webpack-dev-server": "^1.16.2"
```

Comment:
why? In general we need fixed versions (1.16.2) for example. We also don't update libraries if there is no reason.